app/src/main/java/org/mozilla/fenix/home/sessions/SessionBottomSheetFragment.kt

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+   License, v. 2.0. If a copy of the MPL was not distributed with this
+   file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.home.sessions
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import kotlinx.android.extensions.LayoutContainer
+import kotlinx.android.synthetic.main.session_bottom_sheet.view.*
+import org.mozilla.fenix.R
+import org.mozilla.fenix.ext.requireComponents
+import java.lang.IllegalStateException
+
+class SessionBottomSheetFragment : BottomSheetDialogFragment(), LayoutContainer {
+    var archivedSession: ArchivedSession? = null
```

Comment:
I'm glad that you're able to reuse the same bottom sheet here. Please verify that these field variables get set after Save and Restore: 1. Set "Don't Keep Activities". 2. Open the HomeFragment. 3. Open the bottom sheet overflow for a session. 4. Press Home 5. Re-open the app from Recents. 6. Try interacting with the delete and archive buttons. If there's an issue with it, file a minor bug for it because it'll happen when users task switch to/from games and large apps.

Reply:
I’m sorry I meant to give a bit more context for this or but ran out of time before my PT appointment tonight. I definitely want to improve this a bit but this gives us all happy paths for milestone 2. I will open an issue to improve this for the next milestone